dev-workflow-ui/webContent/includes/components/allTasksTable.xhtml

```diff
@@ -26,9 +26,9 @@
     </p:column>
 
     <p:column headerText="Name" filterBy="#{task.name}" sortBy="#{task.name}">
-      <h:commandLink action="#{tasksIvyDevWfBean.redirectToTaskDetails(task.id)}">
+      <h:commandLink action="#{tasksIvyDevWfBean.redirectToTaskDetails(task)}">
         <h:outputText id="taskName" value="#{task.name}" />
-        <p:tooltip for="taskName" value="Task ID: #{task.id}" position="top" />
+        <p:tooltip for="taskName" value="Task: #{task.uuid}" position="top" />
```

Comment:
I think I would leave `Task ID: ...` or `Task: UUID: ...`
i feel like if you only say `Task: ...` and show some weird numbers people might think it should be name and something is broken?
just my thoughts

Reply:
I'll discuss it Bruno, when reviewing the story.